std/net/curl.d

```diff
@@ -2328,11 +2336,20 @@ private bool decodeLineInto(Terminator, Char = char)(ref const(ubyte)[] basesrc,
   *     return len;
   * };
   * http.perform();
+  * ---
+  *
+  */
+
+/**
+  * Tracking progress:
+  *
+  * ---
+  * import std.net.curl, std.stdio;
   *
-  * // Track progress
+  * auto http = HTTP();
   * http.method = HTTP.Method.get;
-  * http.url = "http://upload.wikimedia.org/wikipedia/commons/"
-  *            "5/53/Wikipedia-logo-en-big.png";
+  * http.url = "http://upload.wikimedia.org/wikipedia/commons/" ~
+               "5/53/Wikipedia-logo-en-big.png";
```

Comment:
All lines begin with an asterisk except this one. For consistency it would be good to preserve the comment style.
```suggestion
  *            "5/53/Wikipedia-logo-en-big.png";
```